docs/building_cmake.md

```diff
@@ -0,0 +1,38 @@
+# Building with CMake
+
+It's also possible build this project with cmake.
+
+## Windows with MSVC 2019 64 bits
+
+### Install dependencies
+
+- Download and install openSSL client and headers from [Shining Light Providers](https://slproweb.com/products/Win32OpenSSL.html) (Win64OpenSSL-1_1_1k.msi) or choose another [provider](https://wiki.openssl.org/index.php/Binaries)
+- Download [libasio](https://sourceforge.net/projects/asio/files/asio/1.18.1%20%28Stable%29/asio-1.18.1.zip/download) and uncompress on a new folder.
```

Comment:
Ah, I think I know why you seem to have more difficulties. I usually use vcpkg, seems to me that's a little easier in setup. Maybe you can try this instead of the source downloads (it's from our private documentation from a private repo). That should work for you without changing the root's CMakeLists:



1. Clone https://github.com/microsoft/vcpkg, `master` should suffice
2. Install the deps: `vcpkg.exe install --triplet asio openssl`
3. In VS Code, create the `.vscode/settings.json` and add a top-level
```
"cmake.configureSettings": {
    "CMAKE_TOOLCHAIN_FILE": "C:\\Users\\nilsn\\Documents\\dev\\cpp\\vroom\\vcpkg\\scripts\\buildsystems\\vcpkg.cmake",
},
```
4. Configure and build